docs/source/gbd2019_models/risk_effects/mm_risk_effects/index.rst

```diff
@@ -68,40 +64,40 @@ In Phase 2, some of these risk factors will become correlated with treatment in
     - Source
     - Note
   * - Sex
-    -
-    -
-    - Male
+    - 0.86 (0.68, 1.1)
+    - 0.83 (0.67, 1.03)
     - Female
+    - Male
     - Age, treatment(?)
-    -
-    -
-    -
+    - None 
+    - [van_de_Donk_2018]_
+    - 
   * - Age at diagnosis
-    -
-    -
-    - 65+
-    - <65
+    - 1.30 (1.15, 1.48)
+    - 1.17 (1.05, 1.31)
+    - 70+
+    - <70
     - Sex, treatment
-    -
-    -
+    - None 
+    - [Ailawadhi_2018]_
     -
   * - Renal function at diagnosis
-    -
-    -
+    - 1 (1, 1)
+    - 1 (1, 1)
     - Impaired
     - Not impaired
     - Treatment
-    -
-    -
+    - 
+    - [Dimopoulos_2010]_
     - Impairment defined as eGFR less than 60
```

Comment:
Can you add to this note to explain how there is low data availability at our cutoff, but that existing data suggests no effect?